packages/@react-spectrum/form/chromatic/Form.chromatic.tsx

```diff
@@ -50,6 +54,20 @@ const Template = (): Story<SpectrumFormProps> => (args) => (
       <Radio value="dragons">Dragons</Radio>
     </RadioGroup>
     <TextArea label="Comments" placeholder="How do you feel?" />
+    <ColorField label="Primary Color" />
```

Comment:
lets not add colorfield to forms for chromatic, it's not correct currently i believe

Reply:
It looked pretty good to me, but I'll remove it.